tests/beman/iterator_interface/iterator_interface.test.cpp

```diff
@@ -119,5 +120,38 @@ TEST(IteratorTest, OperatorArrow) {
     ASSERT_EQ(ai->f(), 3);
 }
 
+#ifdef __cpp_explicit_this_parameter
```

Comment:
To capture all compiler+flag configurations where this applies...

```suggestion
#if BEMAN_ITERATOR_INTERFACE_USE_DEDUCING_THIS()
```

Reply:
Removed this by switching to `ext_iterator_interface_compat`